src/Modules/Unix/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="EEFCB906-B326-4E99-9F54-8B4BB6EF3C6D"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="� Microsoft Corporation. All rights reserved."
+Copyright="© Microsoft Corporation. All rights reserved."

```

Comment:
When it looks the same but git shows a diff, it's usually due to a special character change. What I would do in this case is to first revert changes to this file (git checkout), and then open it in a different editor (not vscode), and make the `Get-TimeZone` change.
@iSazonov do you happen to know if there is any configuration file we can use to prevent VSCode from messing up special characters?